app/Resources/doc/changeLog/CHANGELOG-0.2.12.md

```diff
@@ -85,11 +85,14 @@ Url to see changes :
 
  - The front configuration must also be updated. In app/config/routing.yml of your front application, add the following lines :
 
-    open_orchestra_base:
-        resource: "@OpenOrchestraBaseBundle/Resources/config/routing.yml"
-
     open_orchestra_front_preview:
         resource: "@OpenOrchestraFrontBundle/Resources/config/preview_routing.yml"
-  - We have create a `MediaModeBundle` to store the media document, you should activate the bundle : 
+  - We have create a `MediaModeBundle` to store the media document, you should activate the bundle :
```

Comment:
Maybe you should use something more neutral :
'a MediaModeBundle has been created to store ....'